mmv1/products/cloudrun/terraform.yaml

```diff
@@ -143,6 +143,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           project: :PROJECT_NAME
         ignore_read_extra:
           - "autogenerate_revision_name"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "eventarc-basic-tf"
+        primary_resource_id: "default"
```

Comment:
Looks like we need to add `primary_resource_type:` for cgc examples, such as https://github.com/GoogleCloudPlatform/magic-modules/pull/5801/files#diff-00f4a40d3c165c18ec5bd2ef9161cdc3a0715f292f48fbfd17605f9f3d78e1aaR90

Reply:
Hi @shuyama1, thank you for reviewing the code and providing feedback. I had a quick query regarding the feedback: as per my understanding from the https://cloud.devsite.corp.google.com/guides/authoring/terraform/prepare-terraform#add-sample-metadata, the primary_resource_type is needed when the sample needs to use a workaround. The primary resource I selected in this example is the Cloud Run service and as per https://cloud.devsite.corp.google.com/guides/authoring/terraform/cgc-product-tf-yaml#check, I ensured that the Cloud run service is present only between the ####### START datasources ########### and ####### END datasources ########### comments which results in not requiring a workaround as per the doc. So should we add the primary_resource_type field here? Thanks!

Reply:
I think you are absolutely correct! The test is generated successfully! Sorry for the confusion!
The reason that fails the test is actually something else -- let's try using `eventarc_basic_tf` instead of `eventarc-basic-tf` for name, as `-` is not allowed in the test name.

Reply:
No worries! Thank you for helping me with this!